src/Maths/Silk.NET.Maths/Simd/Simd256.GreaterThanOrEqual.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if INTRINSICS
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+#if SSE
+using System.Runtime.Intrinsics.X86;
+#endif
+#if AdvSIMD
+using System.Runtime.Intrinsics.Arm;
+#endif
+
+namespace Silk.NET.Maths
+{
+    public static unsafe partial class Simd256
+    {
+        /// <summary>
+        /// Performs hardware-accelerated GreaterThanOrEqual on 256-bit vectors.
+        /// </summary>
+        [MethodImpl(Scalar.MaxOpt)]
+        public static Vector256<T> GreaterThanOrEqual<T>(Vector256<T> left, Vector256<T> right) where T : unmanaged
+        {
+            return Single(left, right);            
+                        
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector256<T> Single(Vector256<T> left, Vector256<T> right)
+            {
+                if (typeof(T) == typeof(float))
+                {
+#if AVX && NET5_0_OR_GREATER
+                    return Avx2.CompareGreaterThanOrEqual(left.AsSingle(), right.AsSingle()).As<float, T>();
+#elif AVX
+                    return Avx2.Compare(left.AsSingle(), right.AsSingle(), FloatComparisonMode.OrderedGreaterThanOrEqualSignaling).As<float, T>();
```

Comment:
shouldn't this use non-signaling? (`OrderedGreaterThanOrEqualNonSignaling`)

Reply:
I used what [BCL uses](https://source.dot.net/#System.Private.CoreLib/Avx.cs,191). Lmk if you need non-signaling